### ISSUE TITLE
Are you agree to make a kind of merge between our two fork of mcmt

### ISSUE THREAD

**Issue by coco875** [min2222/MCMTForge#34]
I manage to make a mcmt on multiloader (I just need to change the mapping to the official Minecraft), are you agree if I put all of your modifications on my fork ? And some things who can be good is make the continuity of mcmt who are on both modloader discontinued ? (So make the modification your repo becomes cross loader or my repo have your modification and focus both efforts on the same repo)

**Comment by kitovoi111** [comment]
> I manage to make a mcmt on multiloader (I just need to change the mapping to the official Minecraft), are you agree if I put all of your modifications on my fork ? And some things who can be good is make the continuity of mcmt who are on both modloader discontinued ? (So make the modification your repo becomes cross loader or my repo have your modification and focus both efforts on the same repo)

good idea

**Comment by kitovoi111** [comment]
Do you have a world saving error? Otherwise, in this version the world will not be saved at all

**Comment by coco875** [comment]
I have also error when he try to save chunk

**Comment by min2222** [comment]
sure

**Comment by kitovoi111** [comment]
I can’t install MCMT on my server because of the problem of saving the world when the server is stopped.

**Comment by coco875** [comment]
there is also the compat with other mod who need to be check etc (if you want to discuss about how merge and other problem who can happen you can contact me on discord on coco_00 or on the mcmt server)

**Comment by kitovoi111** [comment]
> there is also the compat with other mod who need to be check etc (if you want to discuss about how merge and other problem who can happen you can contact me on discord on coco_00 or on the mcmt server)

Saving the world doesn't work even without mods. So far I have found one problematic mod: Create and IC2 Classic

**Comment by coco875** [comment]
ok ok I will see once I merge or he merge modification and decide with name we keep (net.himeki who heavily inspired this port and where I start or org.jmt, and maybe other choice of name)

**Comment by kitovoi111** [comment]
> ok ok I will see once I merge or he merge modification and decide with name we keep (net.himeki who heavily inspired this port and where I start or org.jmt, and maybe other choice of name)

ok thank you!

**Comment by kitovoi111** [comment]
I have a 200+ online server with mods, so I will be very grateful for your help.

**Comment by kitovoi111** [comment]
![image](https://github.com/min2222/MCMTForge/assets/147491823/ce80e4eb-bc2a-49c5-aa04-7696f72c8641)

**Comment by kitovoi111** [comment]
Now TPS is about 8-17 since the server is on a single thread and optimization can no longer cope, there are too many entities and mechanisms from the Create mod (they are also considered entities)

**Comment by coco875** [comment]
I switch the mapping to mojang mapping https://github.com/coco875/MCMTFabric/tree/1.19-mojmap

**Comment by kitovoi111** [comment]
> I switch the mapping to mojang mapping https://github.com/coco875/MCMTFabric/tree/1.19-mojmap

Already ready to use? Were you able to run Canary, ModernFix, and VMP mods?

**Comment by kitovoi111** [comment]
https://modrinth.com/mod/canary
https://modrinth.com/mod/modernfix
https://modrinth.com/mod/vmp-forge

**Comment by coco875** [comment]
> > I switch the mapping to mojang mapping https://github.com/coco875/MCMTFabric/tree/1.19-mojmap
> 
> 
> 
> Already ready to use? Were you able to run Canary, ModernFix, and VMP mods?

No, I fix nothing I just change the mapping to use the official like this one and bug need to have his own issue so I will say in their issue when it's fixed

**Comment by coco875** [comment]
![image](https://github.com/min2222/MCMTForge/assets/59367621/ac53569b-cc10-4275-86ec-b6d31cc54138)
I succeed to merge https://github.com/coco875/MCMTFabric/tree/1.19-merge (and fix some issue I have with entity and bring trans into the synchronisePlugin

**Comment by coco875** [comment]
(work also on forge)

**Comment by kitovoi111** [comment]
nice

**Comment by kitovoi111** [comment]
Now it is important to make the Canary, VMP, ModernFix mods work

**Comment by coco875** [comment]
Yeah of course but being 2 on the same problem will be simpler to think about compatibility then alone, so I will wait an answer of min2222 and see how we will organize to manage a real continuity to mcmt

**Comment by coco875** [comment]
Don't burn step so complete merge before start another
